pulp_rpm/app/shared_utils.py

```diff
@@ -46,6 +46,11 @@ def is_previous_version(version, target_version):
         bool: True if versions are the same or if the version is older than the target version.
 
     """
+    if version is None or target_version is None:
```

Comment:
doesn't this fix a separate bug?

Reply:
No, we are allowing now null values, so we need to handle them.

I can reference other bug as well if needed, but it's not the reason I'm adding this code here.